gateways/authorizenet/authorizenet.go

```diff
@@ -10,28 +10,26 @@ import (
 	"net/http"
 )
 
-const (
-	baseURL = "https://apitest.authorize.net/xml/v1/request.api"
-)
-
 // AuthorizeNetClient uses merchant name and transaction key to process requests. Optionally can provide custom http clients
 type AuthorizeNetClient struct {
 	merchantName   string
 	transactionKey string
 	httpClient     *http.Client
+	host string
 }
 
 // NewClient uses authentication above with a default http client
-func NewClient(merchantName string, transactionKey string) *AuthorizeNetClient {
-	return NewWithHttpClient(merchantName, transactionKey, common.DefaultHttpClient())
+func NewClient(merchantName string, transactionKey string, environment common.Environment) *AuthorizeNetClient {
```

Comment:
We should standardize whether env is the first or last arg to the `NewClient` functions since we do both right now. Any preference?

Reply:
Ouch yeah thanks for the catch. Lets make it last except for HTTP client?

Reply:
Sounds good.